src/Sulu/Bundle/MediaBundle/Media/Storage/FlysystemStorage.php

```diff
@@ -171,7 +171,7 @@ private function createDirectories(array $storageOptions): void
     public function getPath(array $storageOptions): string
     {
         if (null === $this->rootPath) {
-            return $this->getFilePath($storageOptions);
+            return $this->filesystem->publicUrl($this->getFilePath($storageOptions));
```

Comment:
Local adapter need to return sti the path See 2.6

Reply:
Okay there are now three cases: remote, local, local with prefix path.